install_config/install/disconnected_install.adoc

```diff
@@ -194,16 +200,26 @@ Replace `<tag>` with `{latest-int-tag}` for the latest version.
 endif::[]
 +
 ----
+# docker pull registry.access.redhat.com/openshift3/container-engine:<tag>
+# docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-1-rhel7:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-2-rhel7:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7:<tag>
+# docker pull registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7:<tag>
+# docker pull registry.access.redhat.com/openshift3/logging-auth-proxy:<tag>
+# docker pull registry.access.redhat.com/openshift3/logging-curator:<tag>
 # docker pull registry.access.redhat.com/openshift3/logging-deployer:<tag>
 # docker pull registry.access.redhat.com/openshift3/logging-elasticsearch:<tag>
-# docker pull registry.access.redhat.com/openshift3/logging-kibana:<tag>
 # docker pull registry.access.redhat.com/openshift3/logging-fluentd:<tag>
-# docker pull registry.access.redhat.com/openshift3/logging-curator:<tag>
-# docker pull registry.access.redhat.com/openshift3/logging-auth-proxy:<tag>
+# docker pull registry.access.redhat.com/openshift3/logging-kibana:<tag>
+# docker pull registry.access.redhat.com/openshift3/metrics-cassandra:<tag>
 # docker pull registry.access.redhat.com/openshift3/metrics-deployer:<tag>
 # docker pull registry.access.redhat.com/openshift3/metrics-hawkular-metrics:<tag>
-# docker pull registry.access.redhat.com/openshift3/metrics-cassandra:<tag>
+# docker pull registry.access.redhat.com/openshift3/metrics-hawkular-openshift-agent:<tag>
 # docker pull registry.access.redhat.com/openshift3/metrics-heapster:<tag>
+# docker pull registry.access.redhat.com/openshift3/node:<tag>
```

Comment:
node and openvswitch images should belong to "the required {product-title} containerized components"